distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Dialect.enso

```diff
@@ -182,23 +183,6 @@ type SQLite_Dialect
         _ = statement
         True
 
-    ## PRIVATE
-       Specifies if the Database backend supports WITH clauses in nested queries.
-    supports_nested_with_clause : Boolean
-    supports_nested_with_clause self = True
-
-    ## PRIVATE
-    supports_separate_nan : Boolean
-    supports_separate_nan self = False
```

Comment:
I'm not entirely sure how the Property is better than separate methods like these. To me they are more or less similar level of readability.

It does seem better than the dialect flags / test flags.

Reply:
For me the advantage is just less boiler plate code. It is one line added to dialect rather than 4. Less repeated comments. They are forced to be in one place so you can read them all at once.

I'll admit the advantages are marginal, but I think the new approach is the better of the 2.

Reply:
Sounds good 👍